### PR TITLE
Update day 1 checklist with latest instructions

### PR DESCRIPTION
**Pull request by JohanLorenzo**
I realized the docs were slightly out of date in regard to Ship-It and Balrog.

### CODE REVIEW

File: procedures/release-duty/index.rst [hunk 1]
```diff
@@ -128,11 +128,8 @@ for how to get set up and started with connecting to the VPN.
 How to get Ship-it access
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You need to be added to the `vpn_cloudops_shipit` LDAP group, as well as be added to the allowedlist in the shipit
-config. Both of these can be done by filing a ticket requesting access.
-
-File a Release Engineering bug under the `Applications: Shipit (backend)
-<https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&cloned_bug_id=1626312&product=Release%20Engineering&component=Applications%3A%20Shipit%20%28backend%29>`__
+You just need to be added to the `vpn_cloudops_shipit` LDAP group. File a Release Engineering bug under the `Applications: Shipit (backend)
```
Comment (jcristau) [inline procedures/release-duty/index.rst:131]: nit: remove "just"

Comment (jcristau) [inline procedures/release-duty/index.rst:131]: The bug should live under IT / LDAP, not releng, I think?

**Review by jcristau — CHANGES_REQUESTED** [review]

File: procedures/release-duty/index.rst (hunk 1 above)
Comment (JohanLorenzo) [reply procedures/release-duty/index.rst]: Oops, that's an overlook of mine. Done!

File: procedures/release-duty/index.rst [hunk 2]
```diff
@@ -128,11 +128,8 @@ for how to get set up and started with connecting to the VPN.
 How to get Ship-it access
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You need to be added to the `vpn_cloudops_shipit` LDAP group, as well as be added to the allowedlist in the shipit
-config. Both of these can be done by filing a ticket requesting access.
-
-File a Release Engineering bug under the `Applications: Shipit (backend)
-<https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&cloned_bug_id=1626312&product=Release%20Engineering&component=Applications%3A%20Shipit%20%28backend%29>`__
+You need to be added to the `vpn_cloudops_shipit` LDAP group. File an `Infrastructure & Operations` bug under the `Infrastructure: LDAP
+<https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&cloned_bug_id=1773239&product=Release%20Engineering&component=Applications%3A%20Shipit%20%28backend%29>`__
```
Comment (jcristau) [inline procedures/release-duty/index.rst:132]: the product/component in the link are still off :)

Comment (JohanLorenzo) [reply procedures/release-duty/index.rst]: Oops, there we go!

**Review by jcristau — APPROVED** [review]

**Pull request merged**